tests/extensions/test_pointcloud.py

```diff
@@ -334,3 +337,119 @@ def test_asset_ext_add_to(self) -> None:
+        count_range = RangeSummary(1000, 10000)
+
+        summaries_ext.count = count_range
+
+        self.assertEqual(
+            summaries_ext.count,
+            count_range,
+        )
+
+        summaries_dict = collection.to_dict()["summaries"]
+
+        self.assertEqual(
+            summaries_dict["pc:count"],
+            count_range.to_dict(),
+        )
+
+    def test_type(self) -> None:
+        collection = self.collection.clone()
+        summaries_ext = PointcloudExtension.summaries(collection, True)
+        type_list = [PhenomenologyType.LIDAR, "something"]
+
+        summaries_ext.type = type_list
+
+        self.assertEqual(
+            summaries_ext.type,
+            type_list,
+        )
+
+        summaries_dict = collection.to_dict()["summaries"]
+
+        self.assertEqual(
+            summaries_dict["pc:type"],
+            type_list,
+        )
+
+    def test_encoding(self) -> None:
+        collection = self.collection.clone()
+        summaries_ext = PointcloudExtension.summaries(collection, True)
+        encoding_list = ["LASzip"]
+
+        summaries_ext.encoding = encoding_list
+
+        self.assertEqual(
+            summaries_ext.encoding,
+            encoding_list,
+        )
+
+        summaries_dict = collection.to_dict()["summaries"]
+
+        self.assertEqual(
+            summaries_dict["pc:encoding"],
+            encoding_list,
+        )
+
+    def test_density(self) -> None:
+        collection = self.collection.clone()
+        summaries_ext = PointcloudExtension.summaries(collection, True)
+        density_range = RangeSummary(500.0, 1000.0)
+
+        summaries_ext.density = density_range
+
+        self.assertEqual(
+            summaries_ext.density,
+            density_range,
+        )
+
+        summaries_dict = collection.to_dict()["summaries"]
+
+        self.assertEqual(
+            summaries_dict["pc:density"],
+            density_range.to_dict(),
+        )
+
+    def test_statistics(self) -> None:
+        collection = self.collection.clone()
+        summaries_ext = PointcloudExtension.summaries(collection, True)
+        statistics_list = [
+            Statistic(
+                {
+                    "average": 637294.1783,
+                    "count": 10653336,
+                    "maximum": 639003.73,
+                    "minimum": 635577.79,
+                    "name": "X",
+                    "position": 0,
+                    "stddev": 967.9329805,
+                    "variance": 936894.2548,
+                }
+            )
+        ]
+
+        summaries_ext.statistics = statistics_list
+
+        self.assertEqual(
+            summaries_ext.statistics,
+            statistics_list,
+        )
+
+        summaries_dict = collection.to_dict()["summaries"]
+
+        self.assertEqual(
+            summaries_dict["pc:statistics"],
+            [s.to_dict() for s in statistics_list],
+        )
```

Comment:
@m-mohr @matthewhanson I ended up implementing the `pc:statistics` summaries as a list of [Stats Objects](https://github.com/stac-extensions/pointcloud#stats-object), but I'm not sure if that's right.

Reply:
It seems right to me to have the summary type match the Item type - if the user calculated the stats for a single channel across all Items, e.g. the average elevation of laz asset files in the collection, then that would be represented as a statistic the same way as it would be points in the assets of a single Item